proposals/stage-1/SUPPORTER_PROGRAM/README.md

```diff
@@ -0,0 +1,48 @@
+# Individual Membership Representation Proposal
+>  Stage 1
+
+## Champion
+
+[Sara Chipps](https://github.com/sarajo)
+ 
+## Description
+
+Membership program for OpenJS individual supporters. 
+
+## Required Resources
+
+What resources would be needed to execute it?
+* The current weekly newsletter signup is removed from the OpenJS website. 
+    * Existing subscribers will not be removed
+    * As people sign up to be an individual supporter they will be added to the mailing list
+    * They can request removal at any time
+    * Optional additional email content, such as “What is the CPC and how can I get involved” can be set up via an email drip campaign for new supporters. 
+* Design
+    * OpenJS Member badges, avatars, and swag
```

Comment:
where would badges / avatars lives? Would there be a social network or profile page?

Reply:
I was thinking we could give it to them for their social media profiles/websites. WDYT?

Reply:
Seems reasonable to me.

Before stage 2 we will likely want to make a more in depth proposal about what site that is, and how profiles will work.

Reply:
Yes, that makes lots of sense.